practice-app/sqlfiles/create_tables.sql

```diff
@@ -15,7 +15,7 @@ CREATE TABLE IF NOT EXISTS BookISBNs (
     book_id             INTEGER,
   	isbn                TEXT,
 	PRIMARY KEY(book_id,isbn),
-    FOREIGN KEY(book_id) REFERENCES Books(book_id), -- todo: on delete vsvs
+    FOREIGN KEY(book_id) REFERENCES Books(book_id) ON DELETE CASCADE,
```

Comment:
do we actually need the ON DELETE thing? do you delete books from the  books table?